runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -389,6 +389,10 @@ else if (config.swapLastDestination() && (option.equals("zanaris") || option.equ
 		{
 			swap("last-destination (", option, target, false);
 		}
+		else if (config.swapLastDestinationZanaris() && (option.equals("configure")))
```

Comment:
Since this is just calling the same thing as the if statenement above, can't you just do `if regular || if zanaris`